base/src/main/scala/co/uproot/abandon/Report.scala

```diff
@@ -250,40 +250,72 @@ object Reports {
     }
   }
 
-  def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings): xml.Node = {
-    <abandon>
-       <balance>
-          { state.accState.mkTree(exportSettings.isAccountMatching).toXML }
-       </balance>
-    </abandon>
+  def xmlBalanceExport(state: AppState, exportSettings: XmlExportSettings, withVersion: Boolean): xml.Node = {
+    if (withVersion) {
```

Comment:
To avoid duplicating the code, you can add the `version` attribute after creating the `abandon` node normally.

Example code to add an attribute to an Element:

```
def addAttrib(n: Elem, k: String, v: String) = n % new xml.UnprefixedAttribute(k, v, xml.Null)
```